spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinConfigurationsSenderConfigurationTests.java

```diff
@@ -51,21 +46,9 @@ void shouldSupplyDefaultHttpClientSenderBean() {
 		this.contextRunner.run((context) -> {
 			assertThat(context).hasSingleBean(BytesMessageSender.class);
 			assertThat(context).hasSingleBean(ZipkinHttpClientSender.class);
-			assertThat(context).doesNotHaveBean(URLConnectionSender.class);
 		});
 	}
 
-	@Test
-	void shouldUseUrlConnectionSenderIfHttpClientIsNotAvailable() {
```

Comment:
Instead of removing this test entirely, you can adjust it to verify:

If `HttpClient` is not present, then the `context` should not have `ZipkinHttpClientSender`.

Reply:
I agree.

Reply:
Implemented.